### ISSUE TITLE
User not be able to Change model

### ISSUE THREAD

**Issue by yesbhautik** [mckaywrigley/chatbot-ui#1410]
I have set the global Azure API keys, but still user dont have rights to change model.
Also at initial sign up, user unable to talk with bot, user have to select azure from profile settings, after that user able to talk with BOT with default model only, for sure.
![chrome_KXUavJcJHw](https://github.com/mckaywrigley/chatbot-ui/assets/63995315/8f2d96db-cabf-44d4-89bb-6e608fd86da5)

**Comment by fitchmultz** [comment]
I was able to choose the OpenAI models once I commented out the line `#NEXT_PUBLIC_OLLAMA_URL=http://localhost:11434 # Default` in my .env.local. Give that a try.

I was running into a similar issue. I had entered my OpenAI key and it showed the same message "Unlock models by..." I was able to prompt GPT-4 Turbo, the default selected model, however.

When I entered my MistralAI API key it gave me the option to select those models but the OpenAI models were still not selectable.

Running a hybrid approach with the UI on my Macbook M1 and supabase hosted on supabase.com



![Screenshot 2024-02-09 at 12 43 20 PM](https://github.com/mckaywrigley/chatbot-ui/assets/57411549/3c8653d9-91e8-4b0e-905b-e01cd7026c62)
![Screenshot 2024-02-09 at 12 43 52 PM](https://github.com/mckaywrigley/chatbot-ui/assets/57411549/d6f021fe-4fbb-423b-b188-de88b5493acd)


After commenting out the ollama line I can select all models

![Screenshot 2024-02-09 at 12 48 18 PM](https://github.com/mckaywrigley/chatbot-ui/assets/57411549/fe2a9017-c901-433c-9765-3aaa309d797b)
